src/SonarQube.Bootstrapper/Program.cs

```diff
@@ -22,7 +22,10 @@ namespace SonarQube.Scanner.MSBuild
 {
     internal static class Program
     {
-        private static int Main(string[] args) =>
-            SonarScanner.MSBuild.Program.Main(args);
+        private static int Main(string[] args)
+        {
+            SonarScanner.MSBuild.Program.PrintDeprecatedWarning();
```

Comment:
As discussed offline, _Program.Main_  could be private, and you could have a single public entry point called e.g. _ExecuteFromLegacyEntryPoint(args)_.